inputs/udpjson/udpjson_test.go

```diff
@@ -57,7 +57,7 @@ func TestUdpJson(t *testing.T) {
 	assert.True(t, ok)
 
 	obj = <-events
-	assert.Equal(t, obj["obj2"].(float64), 4)
+	assert.Equal(t, obj["obj2"].(float64), 4.0)
```

Comment:
It's probably the floating point comparison issue. I think a better fix would be something like this:

    assert.InDelta(t, obj["obj2"].float(64), 4.0, 0.01)